### PR TITLE
Fix module name storage

### PR DESCRIPTION
**Pull request by slpopejoy**

### CODE REVIEW

File: test/Chainweb/Test/Pact/Checkpointer.hs [hunk 1]
```diff
@@ -103,6 +110,37 @@ defModule idx = [text| ;;
...
+
+      (CheckpointEnv {..}, SQLiteEnv {..}) <- res
+
+      -- init genesis
+      let hash00 = nullBlockHash
+      void $ _cpRestore _cpeCheckpointer Nothing
+      _cpSave _cpeCheckpointer hash00
+
+      -- block 1: write module records
+      hash01 <- BlockHash <$> liftIO (merkleLogHash "0000000000000000000000000000001a")
```
Comment (larskuhtz) [inline test/Chainweb/Test/Pact/Checkpointer.hs]: There is also an arbitrary instance for `BlockHash` that can be used to get arbitrary block hashes.

For random arbitrary values

```Haskell
hash01 <- generate
```

For deterministic arbitrary values:

```Haskell
import Test.QuickCheck.Gen
import Test.QuickCheck.Random

getArbitrary :: Arbitrary a => Int -> a
getArbitrary seed = unGen arbitrary (mkQCGen 0) seed

-- ...

let hash01 = getArbitrary @BlockHash 0
```

File: test/Chainweb/Test/Utils.hs [hunk 2]
```diff
@@ -816,3 +817,8 @@ runRocks test = withTempRocksDb "chainweb-tests" $ \rdb -> defaultMain (test rdb
 
 runSchedRocks :: (RocksDb -> ScheduledTest) -> IO ()
 runSchedRocks test = withTempRocksDb "chainweb-tests" $ \rdb -> runSched (test rdb)
+
+-- | Convenience to use "-p" with value to match a test run
+-- > matchTest "myTest" $ runSched tests
+matchTest :: String -> IO a -> IO a
+matchTest pat = withArgs ["-p",pat]
```
Comment (larskuhtz) [inline test/Chainweb/Test/Utils.hs]: 👍

File: test/Chainweb/Test/Pact/Checkpointer.hs (hunk 1 above)
Comment (slpopejoy) [reply test/Chainweb/Test/Pact/Checkpointer.hs]: Nice, added this to Chainweb.Test.Utils

File: test/Chainweb/Test/Pact/Checkpointer.hs [hunk 3]
```diff
@@ -103,6 +110,37 @@ defModule idx = [text| ;;
...
+
+      (CheckpointEnv {..}, SQLiteEnv {..}) <- res
+
+      -- init genesis
+      let hash00 = getArbitrary 0
+      void $ _cpRestore _cpeCheckpointer Nothing
+      _cpSave _cpeCheckpointer hash00
+
+      -- block 1: write module records
+      hash01 <- getArbitrary 1
```
Comment (larskuhtz) [inline test/Chainweb/Test/Pact/Checkpointer.hs]:
```suggestion
      let hash01 = getArbitrary 1
```

**Pull request merged**